Makefile

```diff
@@ -48,6 +48,17 @@ endif
 
 ETC_LOCALTIME := $(realpath /etc/localtime)
 
+HOST_ARCH := $(shell arch)
```

Comment:
We need to make this configurable as well to allow cross-compiling the Docker container

Reply:
Done.